src/smartpeak/source/core/SessionHandler.cpp

```diff
@@ -2007,121 +2060,122 @@ namespace SmartPeak
         if (!selected_transitions(i).empty())
           component_names.insert(selected_transitions(i));
       }
-      if (result.conc_fit_data.size() != component_names.size() && result.conc_raw_data.size() != component_names.size())
-      {
-        // LOGD << "Making the calibrators data for plotting";
-        // Update the axis titles and clear the data
-        result.x_axis_title = "Concentration (" + sequence_handler.getSequenceSegments().at(0).getQuantitationMethods().at(0).getConcentrationUnits() + ")";
-        result.y_axis_title = sequence_handler.getSequenceSegments().at(0).getQuantitationMethods().at(0).getFeatureName() + " (au)";
-        result.conc_min = 1e6;
-        result.conc_max = 0;
-        result.feature_min = 1e6;
-        result.feature_max = 0;
-        result.conc_fit_data.clear();
-        result.feature_fit_data.clear();
-        result.conc_raw_data.clear();
-        result.feature_raw_data.clear();
-        result.series_names.clear();
-        for (const auto& sequence_segment : sequence_handler.getSequenceSegments()) {
-          // Extract out raw data used to make the calibrators found in `StandardsConcentrations`
-          std::map<std::string, std::pair<std::vector<float>, std::vector<std::string>>> stand_concs_map; // map of x_data and sample_name for a component
-          for (const auto& stand_concs : sequence_segment.getStandardsConcentrations())
+      // LOGD << "Making the calibrators data for plotting";
+      // Update the axis titles and clear the data
+      result.x_axis_title = "Concentration ratio";
+      result.y_axis_title = sequence_handler.getSequenceSegments().at(0).getQuantitationMethods().at(0).getFeatureName() + " response";
```

Comment:
I would recommend `ratio` instead of response.

Reply:
ok, done.